src/helpers/environment/Environment/Environment.ts

```diff
@@ -24,6 +24,13 @@ export class Environment {
       err: "Please provide an Google Analytics API key in your .env file",
     });
 
+  public static getMultiFarmApiKey = () =>
```

Comment:
I should have removed this function in my last commit but must have forgot to - can follow up after and remove it as it isn't called anywhere.

Reply:
ok thanks ser. Since it's not critical you can follow up with that pr in a future deploy. 🥇